### PR TITLE
[orchagent, SRv6]: mysid support to set default vrf

### PR DESCRIPTION
**Pull request by link19430**
What I did
mysid support to set default vrf

Why I did it
set default vrf not work under local testing

How I verified it
set default vrf of mysid and confirm srv6 works under local testing

Signed-off-by: link_chiang <link_chiang@edge-core.com>

### CODE REVIEW

**Comment by link19430** [comment]
Hi @prsunny
Could you help to review this PR, thanks

**Comment by link19430** [comment]
Hi @prsunny
Could you help to review this PR, thanks

**Comment by prsunny** [comment]
@kperumalbfn , could you please review/sign-off?

File: orchagent/srv6orch.cpp [hunk 1]
```diff
@@ -484,7 +484,11 @@ bool Srv6Orch::createUpdateMysidEntry(string my_sid_string, const string dt_vrf,
     {
         sai_object_id_t dt_vrf_id;
         SWSS_LOG_INFO("DT VRF name %s", dt_vrf.c_str());
-        if (m_vrfOrch->isVRFexists(dt_vrf))
+        if (dt_vrf == "default")
```
Comment (kperumalbfn) [inline orchagent/srv6orch.cpp:487]: @link19430 could you please update test_srv6.py with one mysid entry with default VRF?

**Comment by link19430** [comment]
@kperumalbfn , please help review, thanks.

**Review by kperumalbfn — APPROVED** [review]

**Pull request merged**